test/e2e/manifests/pkg/configuration/digest/configuration.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: configuration-digest
+spec:
+  # NOTE(ezgidemirel): This package is manually built and pushed to the registry.
+  package: xpkg.upbound.io/ezgiorg/ezgi-configuration-getting-started@sha256:de9047e10c479de78c5007fef708d4d21feae8be10b1c748a8276d52e068fcb0
+  ignoreCrossplaneConstraints: true
```

Comment:
Looking that the crossplane.yaml's, we don't need `ignoreCrossplaneConstraints: true`